automation_tools/satellite6/upgrade/tests/utils.py

```diff
@@ -0,0 +1,152 @@
+import os
+import pickle
+import sys
+import time
+
+from automation_tools import manage_daemon
+from automation_tools.bz import bz_bug_is_open
+from automation_tools.satellite6.hammer import (
+    get_attribute_value, hammer,
+    hammer_info_via_name
+)
+from automation_tools.satellite6.upgrade.tasks import (
+    create_rhevm_instance,
+    delete_rhevm_instance,
+    get_rhevm_client,
+    remove_all_docker_containers,
+    wait_till_rhev_instance_status
+)
+from automation_tools.satellite6.upgrade.tasks import docker_execute_command
+from automation_tools.satellite6.upgrade.tools import logger
+from fabric.api import execute, run
+
+rpm1 = 'https://inecas.fedorapeople.org/fakerepos/zoo3/bear-4.1-1.noarch.rpm'
+rpm2 = 'https://inecas.fedorapeople.org/fakerepos/zoo3/camel-0.1-1.noarch.rpm'
+data = {}
+rhev_template = 'sat6-docker-upgrade-template'
+
+logger = logger()
+
+
+def create_dict(dict):
```

Comment:
Also reading the body of this function it seems to be storing the dict data into disk, maybe a better name and a docstring for this function would make easier to get its purpose.